modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -165,7 +165,7 @@ def buildProject(Map options = [:]) {
     lock("$repoName-$NODE_NAME-test") {
       if (hasDatabase()) {
         stage("Set up the database") {
-            runRakeTask("db:drop db:create db:schema:load")
+            runRakeTask("db:drop db:create db:schema:load_if_ruby db:structure:load_if_sql")
```

Comment:
With two votes for `db:reset` I [had a look](https://github.com/search?p=1&q=org%3Aalphagov+db%3Aseed&type=Code&utf8=%E2%9C%93) and most of our rails apps have an empty `db:seed` so it's ultimately not going to affect them if we do run the seeds.  I'll change this to us `db:reset` and we can see if it affects those that do have seeds:

* [specialist publisher](https://github.com/alphagov/specialist-publisher/blob/master/db/seeds.rb) - has a guard against running in envs other than dev, so it almost certainly won't be a problem
* [contacts-admin](https://github.com/alphagov/contacts-admin/blob/master/db/seeds.rb) - this will make a call to the network to import organisations - not sure how well that'll work
* [release](https://github.com/alphagov/release/blob/master/db/seeds.rb)
* [imminence](https://github.com/alphagov/imminence/blob/master/db/seeds.rb) - will load anything in `db/seeds/*.rb` - but there aren't any
* [router-api](https://github.com/alphagov/router-api/blob/master/db/seeds.rb) - there are files in `db/seeds/*.rb` but they seem normal
* [publisher](https://github.com/alphagov/publisher/blob/master/db/seeds.rb) - there's one file in `db/seeds/*.rb` for business support facets which I suspect can maybe be deleted (that's what the only one in imminence was and it's been removed).